src/communication/network_sr.c

```diff
@@ -1131,7 +1132,12 @@ net_server_start (const char *server_name)
       status = -1;
       goto end;
     }
-
+#if defined(SERVER_MODE)
+  if (mmon_initialize (server_name) != NO_ERROR)
+    {
+      goto end;
+    }
+#endif
```

Comment:
```suggestion
  if (mmon_initialize (server_name) != NO_ERROR)
    {
      PRINT_AND_LOG_ERR_MSG ("Failed to initialize memont monitor.\n");
      status = -1;
      goto end;
    }
```
1. I don't think `SERVER_MODE` macro is needed here. Otherwise, correct me.
2. Follow the error handling in this function. 
3. If you want to keep running even though mmon is unavailable for some reason, remove `goto end;` and just print a warning error. (I'm not sure this is required for the mmon)

Reply:
> I don't think SERVER_MODE macro is needed here. Otherwise, correct me.

Among cs,sa,cubrid/CMakelist.txt, only cs has the `${COMMUNICATION_DIR}/network_sr.c`.

Reply:
You're right. I modify it.